project/Dependencies.scala

```diff
@@ -0,0 +1,50 @@
+import sbt._
+
+/** Dependencies for reuse in different parts of the build */
+object Dependencies {
+
+  // NOTE: remember to change the version numbers in the sample projects
+  // when changing them here
+
+  val scalaVersions = Seq("2.11.12", "2.12.8")
+
+  val slf4j = "org.slf4j" % "slf4j-api" % "1.7.25"
+  val typesafeConfig = "com.typesafe" % "config" % "1.3.2"
+  val reactiveStreamsVersion = "1.0.2"
+  val reactiveStreams = "org.reactivestreams" % "reactive-streams" % reactiveStreamsVersion
+  val reactiveStreamsTCK = "org.reactivestreams" % "reactive-streams-tck" % reactiveStreamsVersion
+
+  def mainDependencies = Seq(slf4j, typesafeConfig, reactiveStreams)
+
+  val junit = Seq(
+    "junit" % "junit-dep" % "4.11",
+    "com.novocode" % "junit-interface" % "0.11"
+  )
+  def scalaTestFor(scalaVersion: String) = {
+    val v = "3.0.4"
+    "org.scalatest" %% "scalatest" % v
+  }
+  val logback = "ch.qos.logback" % "logback-classic" % "1.2.3"
+  val hikariCP = "com.zaxxer" % "HikariCP" % "3.2.0"
+
+  val h2 = "com.h2database" % "h2" % "1.4.196"
```

Comment:
In the meantime, this one was update to 1.4.197 in master, but we spotted an issue with 1.4.197 [here](https://github.com/dnvriend/akka-persistence-jdbc/pull/217#issue-246130163) and therefore I'm moving back to 1.4.196.

Reply:
I reverted it back to 1.4.197 (like in master) as we were getting build errors with 1.4.196. 

We can re-evaluate if we get errors like reported [here](https://github.com/dnvriend/akka-persistence-jdbc/pull/217#issue-246130163).